fab/fabfile.py

```diff
@@ -1225,3 +1225,50 @@ def selenium_test():
         'pass': env.jenkins_password,
         'url': url,
     })
+
+
+# good to package all of transifex into a seperate module
+SOURCE_LANGUAGE = "en"
+
+
+@task
+def build_new_translation_source():
+    # if run repeatedly, will keep adding commits
+    # any way to check, before updating?
+    local("./manage.py makemessages -l {source_lang}".format(source_lang=SOURCE_LANGUAGE))
+    local("git add ./locale/{source_lang}/LC_MESSAGES/django.po".format(source_lang=SOURCE_LANGUAGE))
+    local("git commit -m 'Updated translation resource for source language'")
+
+
+@task
+def push_to_transifex():
+    build_new_translation_source()
+    # need to do some checks, so that we are not overriding transifex remote
+    local("tx push --source")
```

Comment:
Good call. I wanted to pull all once and commit before making and pushing new translations. Will add that logic.